### DISCUSSION TITLE
Tell me your most desired holiday destination

### DISCUSSION THREAD

**Discussion by neozoxix** [NAIST-SE/AttractingContributors#1162] [2024 Attraction Contributions]
You can tell me your favorite vacation spot. If possible, you can add reasons why the place is interesting.

link: https://github.com/neozoxix/AttractingContributors-holiday_destination

Thank You

**Comment by 3207-Rhims** [comment]
https://github.com/3207-Rhims/AttractingContributors-Movie , would you be kind enough to contribute to my project ?  I made a PR for you. Thank you

**Comment by neozoxix** [comment]
Dear Rownak,

I cant push my commit, did i have a permission to your repo ?

Thank you

Pada Min, 30 Jun 2024 pukul 22.16 Rownak Borhan Himel <
***@***.***> menulis:

**Reply by 3207-Rhims** [reply]
I think , I my pull request is open, can you try once more ? Thank you https://github.com/NAIST-SE/AttractingContributors/compare/main...3207-Rhims:AttractingContributors-Movie:main?expand=1

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1148

**Comment by Marsall-k** [comment]
I just made a PR on your Repo.
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon
